carsus/conftest.py

```diff
@@ -72,6 +72,18 @@ def pytest_addoption(parser):
                      help="filename for the testing database")
     parser.addoption("--refdata", dest='refdata', default=None,
                      help="carsus-refdata folder location")
+    
+@pytest.fixture
+def withtestdb(request):
+    return request.config.getoption("--test-db")
+
+@pytest.fixture
+def withrefdata(request):
+    return request.config.getoption("--refdata")
+
+@pytest.fixture
+def runslow(request):
+    return request.config.getoption("--runslow")
```

Comment:
I read somewhere that importing fixtures for contfest is generally not a good practice. Should we be moving markers to`pytest.ini`? What do you think?

Reply:
Well these aren't working very well regardless. For the cmfgen tests they are not correctly skipping if `--refdata` is not part of the pytest command. If you want to take a look and modify, please do so.

Reply:
I would look into this. Should I create a separate PR then?

Reply:
I think you should just modify this PR.

Reply:
sure